crates/sui-framework/sources/test/random.move

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module sui::random {
```

Comment:
sui::test_random, just in case we will want to use this name for a non test module